### ISSUE TITLE
help me 

### ISSUE THREAD

**Issue by jaylinlucero** [chainapsis/keplr-wallet#1119]
tell me on which derivative the Dymension address coins are generated

on standard m/44'/118' other addresses are obtained

**Comment by HeesungB** [comment]
@jaylinlucero Hi, currently Keplr is generating Dymension addresses with the path m/44'/60'.
Any idea what could be the problem?

**Comment by jaylinlucero** [comment]
Yes, it’s very strange, I’m trying to generate it using the same path... the address is still incorrect..

**Comment by HeesungB** [comment]
@jaylinlucero Can you elaborate a little bit on what's going on?

**Comment by jaylinlucero** [comment]
Yes, I just can’t make an address generator from a mnemonic phrase, maybe you can give an example of a script to generate a DYM coin address from a mnemonic phrase that would correspond to what the Keplr cryptocurrency wallet uses

> @jaylinlucero Can you elaborate a little bit on what's going on?

**Comment by jaylinlucero** [comment]
> @jaylinlucero Can you elaborate a little bit on what's going on?

you help me?

**Comment by Sm4rty-1** [comment]
I am also facing the same issue. Did you get any solutions? @jaylinlucero 

Could you please help @jaylinlucero ?
